lib/attr_encrypted.rb

```diff
@@ -303,7 +303,7 @@ def encrypted_attributes
   #   User.encrypt_email('SOME_ENCRYPTED_EMAIL_STRING')
   def method_missing(method, *arguments, &block)
     if method.to_s =~ /^((en|de)crypt)_(.+)$/ && attr_encrypted?($3)
-      send($1, $3, *arguments)
+      send("attr_encrypted_#{$1}", $3, *arguments)
```

Comment:
Fixed this to call the new methods for encrypt/decrypt provided by attr_encrypted